README.md

```diff
@@ -0,0 +1,10 @@
+# movies-series SQL analysis
+Analyzing movies and series dataset to answer real life questions.
+# Description
+The purpose of this SQL project was to analyze movies and series from netflix data.
+To achieve the project's goals, a series of SQL queries were written to extract and analyze relevant data. These queries included aggregations to identify the most popular movies or series, as well as joins.
+# How to install
+1. Install to your computer PostgreSQL.
+2. Go to Kaggle website and search for: Latest Netflix data with 26+ joined attributes, downloade the csv to your computer.
+3. Open postgrsql, tap on "new table" and downloade the netflix data.
+4. Go to Github, copy the queries and run them in postgresql.
```

Comment:
In real life - others will clone the repository to their local machines - so maybe its better to make this step generic - something like:

```suggestion
4. Run the queries in your postgresql client of choice.
```